src/api/artifact/model.go

```diff
@@ -82,14 +84,27 @@ func (a *Artifact) ToSwagger() *models.Artifact {
 			Href:     link.HREF,
 		}
 	}
+	for _, label := range a.Labels {
+		art.Labels = append(art.Labels, &models.Label{
```

Comment:
Should the labels in the artifact be a full one (with all properties)?

Reply:
Yes, almost all of the properties is needed when rendering the UI

Reply:
that's weird